app/SharedWebComponents/Components/Answer.razor

```diff
@@ -28,11 +28,11 @@
                                 }
                             </div>
                         }
-                        @if (answer is { FollowupQuestions.Count: > 0 })
+                        @if (Retort?.Context?.FollowupQuestions is { Length: > 0 })
```

Comment:
Why this part need to be changed

Reply:
To be honest, I'm not sure either. Based on my testing, the answer only works for non-streaming cases.

I spent several hours debugging but couldn't figure out why. Then, I tried making the FollowupQuestions section similar to other parts (e.g., Supporting Content) in this file by using Retort.Context, and it worked normally.

If you familiar with this, can let me know as well.